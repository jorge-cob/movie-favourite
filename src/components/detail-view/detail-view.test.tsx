import React from 'react';
import { shallow } from 'enzyme';
import DetailView from './detail-view';

const mockMovie = {
  id: '522',
  title: 'Titanic',
  release_date: '1991-04-1',
  vote_average: '7.8',
  poster_path: 'http://fakeroute.com',
  overview: 'Wow, Jack, dont sink!',
};

it('should render DetailView component', () => {
  expect.assertions(1);
  const component = shallow(<DetailView movie={mockMovie} />);  
  expect(component).toMatchSnapshot();
});
