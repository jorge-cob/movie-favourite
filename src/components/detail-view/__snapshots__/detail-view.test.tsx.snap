// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render DetailView component 1`] = `ShallowWrapper {}`;
